data/mods/Magiclysm/Spells/attunements/Alchemist.json

```diff
@@ -0,0 +1,82 @@
+[
+  {
+    "id": "alchemist_make_potion_bomb",
+    "type": "SPELL",
+    "name": "Manafusion bomb",
+    "description": "Oversaturates potion starter with mana to create a deadly mana explosion.",
+    "valid_targets": [ "none" ],
+    "flags": [ "SOMATIC", "WONDER" ],
+    "effect": "recover_energy",
+    "effect_str": "STAMINA",
+    "//": "I almost forgot how i make this spell, so: you cast spell, main one give you a little of stamina, and call secondary spell that create bomb. I make it like that to avoid bug with activated items and timer mechanic incompability and fact that PERMANENT flag works only on max lvl spell.",
+    "components": "spell_components_manabomb",
+    "shape": "blast",
+    "min_damage": 1,
+    "max_damage": 3,
+    "damage_increment": 0.07,
```

Comment:
what's the point in these numbers? you have 10,000 stamina when your stamina bar is full

Reply:
again, its for spawning more than 1 bomb using wonder mechanic (3 bombs at max)